app/auth/mutations/signup.test.ts

```diff
@@ -0,0 +1,65 @@
+import { hash256, Ctx } from "blitz"
+import db from "db"
+import signup from "../queries/signup"
+
+beforeEach(async () => {
+  await db.$reset()
+})
+
+describe("signup queue", () => {
+  it("does throw error if token doesn't exist", async () => {
+    // use not existing token
+    await expect(signup({ token: "plain-token" }, {} as Ctx)).resolves.toBe(null)
```

Comment:
why should it throw NULL and not an error?

Reply:
Signup returns the created user as an object. If no user could be created (because of an invalid token) the query returns `NULL`. The question  is: whether we want an error to be thrown or not...the result is the same.